apps/web/src/pages/templates/components/templates-store/TemplatesStoreModal.tsx

```diff
@@ -106,6 +106,7 @@ export const TemplatesStoreModal = ({ general, isOpened, onClose }: ITemplatesSt
           <CanvasHolder>
             <ReactFlowProvider>
               <FlowEditor
+                key={selectedTemplate._id}
```

Comment:
this way we can update the Workflow Editor position when another blueprint template is selected

Reply:
If the `selectedTemplate._id` has changed, the `rerender` aka `reconciliation` algorithm will unmount `FlowEditor` and mount it again because the `key` prop has changed. 
Basically, when React does rerender the same element in the tree it checks if the type is the same, or the key is the same, if any of these changes it does unmount for the old component, and mounts for the new one.